### PR TITLE
Add RELEASE.md

### PR DESCRIPTION
**Pull request by marquiz**
Describe the release process of cri-resource-manager.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Release Process
+
+The process to release a new version of cri-resource-manager is:
+
+- [ ] File [a new issue](https://github.com/intel/cri-resource-manager/issues/new)
+  to propose a new release. Copy this checklist into the issue description.
+- [ ] In the issue description, add a changelog section, describing changes
+  since the last release.
+- [ ] Run `git tag -a -m "CRI Resource Manager release $VERSION" $VERSION`.
```
Comment (klihub) [inline RELEASE.md:9]: I would suggest adding
```
 - [ ] run `make release-tests` to run an extended set of tests prior to a release
```
And add already now an empty `release-tests` target to our `Makefile` which we then start
populating with things like running end-to-end test, etc. as we go/get them up and running.

**Comment by klihub** [comment]
Looks good, all of these make sense to me.

File: RELEASE.md (hunk 1 above)
Comment (klihub) [reply RELEASE.md:9]: Should we add steps covering our internally mandated other checks prior to a release ?

**Review by klihub — DISMISSED** [review]

**Comment by klihub** [comment]
One more question: #386 proposes to add INSTALL.md as `docs/INSTALL.md` which is then referenced in `README.md`.
Should we put both in `docs`, or is `RELEASE.md` considered more of a self-contained entity which is not referenced from
anywhere else in the documentation ?

File: RELEASE.md (hunk 1 above)
Comment (marquiz) [reply RELEASE.md:9]: Yeah, I'll add these 👍

**Comment by marquiz** [comment]
> Should we put both in `docs`, or is `RELEASE.md` considered more of a self-contained entity which is not referenced from
> anywhere else in the documentation ?

I personally probably see `RELEASE.md` as maintainer aid and not so much part of the user/developer documentation. Thus, I'd just probably keep it in the root alongside `README.md`. I don't have any strong opinion on this, though, so we can move it under docs if you think that would be more appropriate.

**Comment by klihub** [comment]
> > Should we put both in `docs`, or is `RELEASE.md` considered more of a self-contained entity which is not referenced from
> > anywhere else in the documentation ?
> 
> I personally probably see `RELEASE.md` as maintainer aid and not so much part of the user/developer documentation. Thus, I'd just probably keep it in the root alongside `README.md`. I don't have any strong opinion on this, though, so we can move it under docs if you think that would be more appropriate.

Ok. Makes sense. We can leave it where it is.

File: RELEASE.md (hunk 1 above)
Comment (marquiz) [reply RELEASE.md:11]: Added both

**Comment by marquiz** [comment]
> Ok. Makes sense. We can leave it where it is.

K, and we can always move it later if we change our mind.

Any comments on the latest (indented) version?

**Review by klihub — APPROVED** [review]
LGTM.

**Pull request merged**